helm/cluster-cloud-director/templates/_helpers.tpl

```diff
@@ -135,16 +135,6 @@ files:
 {{- include "staticRoutes" . | nindent 2}}
 {{- end }}
 
-preKubeadmCommands:
```

Comment:
Why remove these?

Reply:
They're not used, the same commands are already in the template.

Reply:
They're in [kubeadmcontrolplane](https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml#L125) but not in [kubeadmconfigtemplate](https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/templates/kubeadmconfigtemplate.yaml#L13)